recipes/Vose-Alias-Method/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "Vose-Alias-Method" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "6ddc19008b917cbe46f3b2251ea0a5588f2261ad7b6f4f827285b47f9e099040"
+
+build:
+  number: 0
```

Could it be ``noarch``?
```suggestion
  number: 0
  noarch: python
```